tests/bettertransformer/test_bettertransformer_encoder.py

```diff
@@ -44,6 +44,8 @@
     "hf-internal-testing/tiny-random-MarkupLMModel",
     "hf-internal-testing/tiny-random-BertModel",
     "ybelkada/random-tiny-BertGenerationModel",
+    "hf-internal-testing/tiny-random-MBartModel",
+    "hf-internal-testing/tiny-random-nllb",
```

Comment:
I think that you have to move them to `ALL_ENCODER_DECODER_MODELS`: the test `pytest tests/bettertransformer/test_bettertransformer_encoder.py::BetterTransformersEncoderDecoderTest` will only run on the models listed on `ALL_ENCODER_DECODER_MODELS` ;)